discovery/README.md

```diff
@@ -138,8 +138,6 @@ $ swarm list zk://<zookeeper_addr1>,<zookeeper_addr2>/<path>
 ```bash
 # start the manager on any machine or your laptop
 $ swarm manage -H <swarm_ip:swarm_port> nodes://<node_ip1:2375>,<node_ip2:2375>
-#or
-$ swarm manage -H <swarm_ip:swarm_port> nodes://<node_ip1:2375>,<node_ip2:2375>
```

Comment:
here it should be 
`$ swarm manage -H <swarm_ip:swarm_port> <node_ip1:2375>,<node_ip2:2375>`